pool/paymentmgr.go

```diff
@@ -415,11 +415,25 @@ func (pm *PaymentMgr) isPaymentRequested(accountID string) bool {
 	return ok
 }
 
-// addPaymentRequest creates a payment request from the provided account.
-func (pm *PaymentMgr) addPaymentRequest(accountID string) {
+// addPaymentRequest creates a payment request from the provided account
+// if not already requested.
+func (pm *PaymentMgr) addPaymentRequest(addr string) error {
+	_, err := dcrutil.DecodeAddress(addr, pm.cfg.ActiveNet)
```

Comment:
There are three places where `Account()` is preceeded by a call to `DecodeAddress()`.

Perhaps `DecodeAddress` can be moved inside of `Account()`? It would remove duplication and ensure the a new caller of `Account` doesnt forget to call `DecodeAddress` first

Reply:
Good catch, rectifying.